src/controllers/buildingsController.js

```diff
@@ -13,27 +11,32 @@ const getAllBuildings = async (req, res) => {
   }
 };
 
-const getBuildingsByAddress = address => buildings.filter(building => building.address === address);
-
-const getBuildingById = id => {
-  const building = buildings.find(buildingItem => buildingItem.id.toString() === id);
-  return building;
-};
-
-const deleteBuildingById = id => {
-  buildings = buildings.filter(building => building.id.toString() !== id);
-  fs.writeFileSync(
-    `${__dirname}/../data/buildings.json`,
-    JSON.stringify(buildings),
-    { encoding: 'utf8', flag: 'w' },
-  );
+const createBuilding = async (req, res) => {
+  if (!req.body.name || !req.body.phone || !req.body.address || !req.body.boilers) {
+    return res.status(500).json({
+      msg: 'Missing required fields to create a building'
+    });
+  }
 
-  return buildings;
+  const building = new models.Building({
+    name: req.body.name,
+    address: req.body.addres,
```

Comment:
```suggestion
    address: req.body.address,
```
Please, take a look at the buildings schema, you defined `adress` instead of `address`